topics/lwdita-input.dita

```diff
@@ -7,11 +7,27 @@
   <titlealts>
     <navtitle>Lightweight DITA</navtitle>
   </titlealts>
-  <shortdesc>DITA-OT provides preview support for the MDITA and HDITA authoring formats proposed for
+  <shortdesc>DITA-OT provides preview support for the XDITA, MDITA and HDITA authoring formats proposed for
     <xref keyref="lwdita"/>, or “<term>LwDITA</term>”. These proposed formats are alternative representations of DITA
-    content in Markdown or HTML.</shortdesc>
+    content in XML, Markdown and HTML5.</shortdesc>
 
   <body>
+    <section>
+      <title>XDITA</title>
+      <p>XDITA is the LwDITA authoring format based on XML. It is much simpler than full DITA 1.3 XML. There is an XDITA
+        map and an XDITA topic.</p>
+      <p>The XDITA parser included in the <codeph>org.lwdita</codeph> plug-in provides preliminary support for
+        XDITA.</p>
+      <p>The <xmlatt>format</xmlatt> attribute on <xmlelement>topicref</xmlelement> in a map can be set to
+          <codeph>xdita</codeph> to apply LwDITA-specific processing to XDITA topics:</p>
+      <p>
+        <codeblock outputclass="normalize-space show-line-numbers show-whitespace">&lt;map>
+  &lt;topicref href="xdita-topic.xml" <b>format="xdita"</b>/>
+&lt;/map></codeblock>
+      </p>
+      <note>You can use an XDITA map or a full DITA 1.3 map for XDITA, MDITA and HDITA topics. See the sample LwDITA
+        files in <filepath>dita-ot/plugins/org.oasis-open.xdita.v0_2_1/samples</filepath>.</note>
```

Comment:
The latest plug-in version (that will ship with DITA-OT 3.2) is `org.oasis-open.xdita.v0_2_2`, but I'll merge this as is and take it from here.